src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/TestFixtures/SQLSetupStrategyCertStoreProvider.cs

```diff
@@ -29,6 +29,9 @@ internal override void SetupDatabase()
             databaseObjects.AddRange(columnEncryptionKeys);
 
             List<Table> tables = CreateTables(columnEncryptionKeys);
+
+            Dictionary<string, string> truncationTableNames = sqlBulkTruncationTableNames;
```

Comment:
Where is this `truncationTableNames` used?

Reply:
It was added by mistake removed.